src/chime_dash/app/pages/sidebar.py

```diff
@@ -172,7 +204,25 @@ def __init__(self, language, defaults):
             dom_updates=OrderedDict(pars="children"),
             callback_fn=Sidebar.update_parameters,
         )
-        super().__init__(language, defaults, [input_change_callback])
+        input_store_callback = ChimeCallback(
```

Comment:
I think for simplicity and readability I'd follow the same pattern followed for `load_params_callback` and call this `update_store_callback` (according to the callback function name).